From 4143747efee3d5153c1298ecbffdb2692ddcee1d Mon Sep 17 00:00:00 2001
From: Torben Hohn <torbenh@linutronix.de>
Date: Tue, 13 Oct 2015 16:30:48 +0200
Subject: [PATCH 5/5] packaging builddeb: add busybox to kernel dependency

This is to prevent broken initramfs where busybox is missing

Signed-off-by: Torben Hohn <torbenh@linutronix.de>
---
 scripts/package/builddeb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/package/builddeb b/scripts/package/builddeb
index 89cf404..c22e512 100755
--- a/scripts/package/builddeb
+++ b/scripts/package/builddeb
@@ -397,7 +397,7 @@ EOF
 		cat <<EOF >> debian/control
 
 Package: $packagename
-Depends: kmod | module-init-tools, linux-base (>= 3~), debconf (>= 0.5) | debconf-2.0, initramfs-tools (>= 0.99~) | linux-initramfs-tool
+Depends: kmod | module-init-tools, linux-base (>= 3~), debconf (>= 0.5) | debconf-2.0, initramfs-tools (>= 0.99~) | linux-initramfs-tool, busybox
 Pre-Depends: debconf | debconf-2.0
 Provides: linux-image, linux-image-2.6, linux-modules-$version
 Suggests: $fwpackagename
-- 
2.1.4

